<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="AMICHAITO1-HHS" ignored="1" name="Automated test - End2End" tests="11" failures="8" timestamp="13 Jun 2019 14:07:53 GMT" time="254.743" errors="0">
  <testcase name="initialOfferQuestions" time="19.671" classname="TestNG.sanityAllOptions"/>
  <testcase name="initialOfferScreen" time="25.067" classname="TestNG.sanityAllOptions"/>
  <testcase name="finalOfferQuestions" time="35.746" classname="TestNG.sanityAllOptions"/>
  <testcase name="finalOfferScreen" time="30.436" classname="TestNG.sanityAllOptions">
    <failure type="java.lang.Exception" message="Failed in FINAL OFFER screen">
      <![CDATA[java.lang.Exception: Failed in FINAL OFFER screen
at TestNG.sanityAllOptions.finalOfferScreen(sanityAllOptions.java:679)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- finalOfferScreen -->
  <testcase name="covers" time="30.478" classname="TestNG.sanityAllOptions">
    <failure type="java.lang.Exception" message="Failed in COVERS screen">
      <![CDATA[java.lang.Exception: Failed in COVERS screen
at TestNG.sanityAllOptions.covers(sanityAllOptions.java:762)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- covers -->
  <testcase name="payments" time="31.358" classname="TestNG.sanityAllOptions">
    <failure type="java.lang.Exception" message="Failed in PAYMENTS screen">
      <![CDATA[java.lang.Exception: Failed in PAYMENTS screen
at TestNG.sanityAllOptions.payments(sanityAllOptions.java:814)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- payments -->
  <testcase name="returnSteps" time="0.032" classname="TestNG.sanityAllOptions">
    <failure type="java.lang.Exception" message="Failed in moveing BACKWARD">
      <![CDATA[java.lang.Exception: Failed in moveing BACKWARD
at TestNG.sanityAllOptions.returnSteps(sanityAllOptions.java:917)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- returnSteps -->
  <testcase name="forwardSteps" time="2.02" classname="TestNG.sanityAllOptions">
    <failure type="java.lang.Exception" message="Failed in moveing FORWARD">
      <![CDATA[java.lang.Exception: Failed in moveing FORWARD
at TestNG.sanityAllOptions.forwardSteps(sanityAllOptions.java:1005)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- forwardSteps -->
  <testcase name="iframeCreditGuard" time="40.4" classname="TestNG.sanityAllOptions">
    <failure type="java.lang.Exception" message="Failed in Iframe CreditGuard">
      <![CDATA[java.lang.Exception: Failed in Iframe CreditGuard
at TestNG.sanityAllOptions.iframeCreditGuard(sanityAllOptions.java:1054)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- iframeCreditGuard -->
  <testcase name="thankYouPage" time="35.482" classname="TestNG.sanityAllOptions">
    <failure type="java.lang.Exception" message="Failed to display thank you page">
      <![CDATA[java.lang.Exception: Failed to display thank you page
at TestNG.sanityAllOptions.thankYouPage(sanityAllOptions.java:1071)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- thankYouPage -->
  <testcase name="registeredAccount" time="4.019" classname="TestNG.sanityAllOptions">
    <failure type="java.lang.Exception" message="Failed to display thank you page">
      <![CDATA[java.lang.Exception: Failed to display thank you page
at TestNG.sanityAllOptions.registeredAccount(sanityAllOptions.java:1092)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- registeredAccount -->
  <testcase name="callMeWizard2" time="0.0" classname="TestNG.sanityAllOptions">
    <ignored/>
  </testcase> <!-- callMeWizard2 -->
</testsuite> <!-- Automated test - End2End -->
